### PR TITLE
Added documentation for the »multipart/form-data« feature

### PR DESCRIPTION
**Pull request by lapistano**
Corresponding to https://issue.swisscom.ch/browse/EVO-771

### CODE REVIEW

File: source/_pages/api/file.md [hunk 1]
```diff
@@ -82,6 +82,23 @@ curl -v -X PUT -H "Content-Type: text/plain" \
...
+POST request.
+Since this is basically a form submit the information is send as form fields:
+- *metadata* » use for the metadata formerly sent as payload in step 2
+- *upload* » use to send the file to be stored
+
+```bash
+curl -X POST \
+     -F 'metadata={"action":[{"command":"print"},{"command":"archive"}]}' \
+     -F upload=@test.txt \
+     http://graviton.nova.scapp.io/file
```
Comment (hairmare) [inline source/_pages/api/file.md]: Please use `https://exmaple.org` for examples.

Comment (lapistano) [inline source/_pages/api/file.md]: ok, I will replace every related url then. agreed?

Comment (hairmare) [inline source/_pages/api/file.md]: Yes please. Most of them were probably already changed in #13.

File: source/_pages/api/file.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Create and upload a simple text file as follows.
 echo "Hallo Graviton" > test.txt
 
 curl -v -X POST -H "Content-Type: text/plain" \
-    -T'{test.txt}' https://example.org/file/
+    -T'{test.txt}' https://exmaple.org/file
```
Comment (q-src) [inline source/_pages/api/file.md]: typo 'exmaple' instead of 'example', please check also the exmaples below ;-)

**Pull request merged**